### YamlMime:ManagedReference
items:
- uid: com.microsoft.azure.management.graphrbac._graph_error_exception
  id: _graph_error_exception
  parent: com.microsoft.azure.management.graphrbac
  children:
  - com.microsoft.azure.management.graphrbac._graph_error_exception.body()
  - com.microsoft.azure.management.graphrbac._graph_error_exception.GraphErrorException(final String,final Response<ResponseBody>)
  - com.microsoft.azure.management.graphrbac._graph_error_exception.GraphErrorException(final String,final Response<ResponseBody>,final GraphError)
  href: com.microsoft.azure.management.graphrbac._graph_error_exception.yml
  langs:
  - java
  name: GraphErrorException
  nameWithType: GraphErrorException
  fullName: com.microsoft.azure.management.graphrbac.GraphErrorException
  type: Class
  source:
    remote: &o0
      path: azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/GraphErrorException.java
      branch: 880e2022bf46b71f481107effb7c878a9ae065dd
      repo: https://github.com/Azure/azure-sdk-for-java
    path: azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/GraphErrorException.java
    startLine: 17
  package: com.microsoft.azure.management.graphrbac
  summary: <p>Exception thrown for an invalid response with <xref uid="com.microsoft.azure.management.graphrbac._graph_error" data-throw-if-not-resolved="false">GraphError</xref> information. </p>
  syntax: &o1
    content: public class GraphErrorException
  inheritance:
  - java.lang.Object
  - RestException
- uid: com.microsoft.azure.management.graphrbac._graph_error_exception.body()
  id: body()
  parent: com.microsoft.azure.management.graphrbac._graph_error_exception
  href: com.microsoft.azure.management.graphrbac._graph_error_exception.yml
  langs:
  - java
  name: body()
  nameWithType: GraphErrorException.body()
  fullName: GraphError com.microsoft.azure.management.graphrbac.GraphErrorException.body()
  overload: com.microsoft.azure.management.graphrbac._graph_error_exception.body*
  type: Method
  source:
    remote: *o0
    path: azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/GraphErrorException.java
    startLine: 40
  package: com.microsoft.azure.management.graphrbac
  syntax:
    content: public GraphError body()
    return:
      type: com.microsoft.azure.management.graphrbac._graph_error
- uid: com.microsoft.azure.management.graphrbac._graph_error_exception.GraphErrorException(final String,final Response<ResponseBody>)
  id: GraphErrorException(final String,final Response<ResponseBody>)
  parent: com.microsoft.azure.management.graphrbac._graph_error_exception
  href: com.microsoft.azure.management.graphrbac._graph_error_exception.yml
  langs:
  - java
  name: GraphErrorException(final String message, final Response<ResponseBody> response)
  nameWithType: GraphErrorException.GraphErrorException(final String message, final Response<ResponseBody> response)
  fullName: com.microsoft.azure.management.graphrbac.GraphErrorException.GraphErrorException(final String message, final Response<ResponseBody> response)
  overload: com.microsoft.azure.management.graphrbac._graph_error_exception.GraphErrorException*
  type: Constructor
  source:
    remote: *o0
    path: azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/GraphErrorException.java
    startLine: 24
  package: com.microsoft.azure.management.graphrbac
  summary: >-
    <p>Initializes a new instance of the <xref uid="com.microsoft.azure.management.graphrbac._graph_error_exception" data-throw-if-not-resolved="false">GraphErrorException</xref> class.</p>

    <p></p>
  syntax:
    content: public GraphErrorException(final String message, final Response<ResponseBody> response)
    parameters:
    - id: message
      type: 3d93591e
      description: <p>the exception message or the response content if a message is not available </p>
    - id: response
      type: 4f402e61
      description: <p>the HTTP response </p>
- uid: com.microsoft.azure.management.graphrbac._graph_error_exception.GraphErrorException(final String,final Response<ResponseBody>,final GraphError)
  id: GraphErrorException(final String,final Response<ResponseBody>,final GraphError)
  parent: com.microsoft.azure.management.graphrbac._graph_error_exception
  href: com.microsoft.azure.management.graphrbac._graph_error_exception.yml
  langs:
  - java
  name: GraphErrorException(final String message, final Response<ResponseBody> response, final GraphError body)
  nameWithType: GraphErrorException.GraphErrorException(final String message, final Response<ResponseBody> response, final GraphError body)
  fullName: com.microsoft.azure.management.graphrbac.GraphErrorException.GraphErrorException(final String message, final Response<ResponseBody> response, final GraphError body)
  overload: com.microsoft.azure.management.graphrbac._graph_error_exception.GraphErrorException*
  type: Constructor
  source:
    remote: *o0
    path: azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/GraphErrorException.java
    startLine: 35
  package: com.microsoft.azure.management.graphrbac
  summary: >-
    <p>Initializes a new instance of the <xref uid="com.microsoft.azure.management.graphrbac._graph_error_exception" data-throw-if-not-resolved="false">GraphErrorException</xref> class.</p>

    <p></p>
  syntax:
    content: public GraphErrorException(final String message, final Response<ResponseBody> response, final GraphError body)
    parameters:
    - id: message
      type: 3d93591e
      description: <p>the exception message or the response content if a message is not available </p>
    - id: response
      type: 4f402e61
      description: <p>the HTTP response </p>
    - id: body
      type: 93de0ef9com.microsoft.azure.management.graphrbac._graph_error
      description: <p>the deserialized response body </p>
references:
- uid: 3d93591e
  spec.java:
  - name: final String
    fullName: final String
- uid: 4f402e61
  spec.java:
  - name: final Response<ResponseBody>
    fullName: final Response<ResponseBody>
- uid: com.microsoft.azure.management.graphrbac._graph_error_exception.GraphErrorException*
  name: GraphErrorException
  nameWithType: GraphErrorException.GraphErrorException
  fullName: com.microsoft.azure.management.graphrbac.GraphErrorException.GraphErrorException
  package: com.microsoft.azure.management.graphrbac
- uid: 93de0ef9com.microsoft.azure.management.graphrbac._graph_error
  spec.java:
  - name: 'final '
    fullName: 'final '
  - uid: com.microsoft.azure.management.graphrbac._graph_error
    name: GraphError
    fullName: com.microsoft.azure.management.graphrbac.GraphError
    href: com.microsoft.azure.management.graphrbac._graph_error.yml
- uid: com.microsoft.azure.management.graphrbac._graph_error_exception.body*
  name: body
  nameWithType: GraphErrorException.body
  fullName: GraphError com.microsoft.azure.management.graphrbac.GraphErrorException.body
  package: com.microsoft.azure.management.graphrbac
- uid: com.microsoft.azure.management.graphrbac._graph_error_exception
  parent: com.microsoft.azure.management.graphrbac
  href: com.microsoft.azure.management.graphrbac._graph_error_exception.yml
  name: GraphErrorException
  nameWithType: GraphErrorException
  fullName: com.microsoft.azure.management.graphrbac.GraphErrorException
  type: Class
  summary: <p>Exception thrown for an invalid response with <xref uid="com.microsoft.azure.management.graphrbac._graph_error" data-throw-if-not-resolved="false">GraphError</xref> information. </p>
  syntax: *o1
- uid: com.microsoft.azure.management.graphrbac._graph_error
  parent: com.microsoft.azure.management.graphrbac
  href: com.microsoft.azure.management.graphrbac._graph_error.yml
  name: GraphError
  nameWithType: GraphError
  fullName: com.microsoft.azure.management.graphrbac.GraphError
  type: Class
  summary: <p>Active Directory error information. </p>
  syntax:
    content: public class GraphError
